language-design.md

```diff
@@ -17,6 +17,27 @@ For each quote, describe what it was about the quote that led you pick it.
 
 **Response**
 
+"Next to English, all computer programming languages are small; as we write code, we must stop now and then
+to define some new term that we will need to use in more than one place" [Steele, 1998].
+- This quote really made me think about the speakers objective in giving this talk. I had never really comapered
+programming languages to the English language because in my mind they were so different from each other. Once the 
+speaker presented this idea, I really stopped to think about how small and less complex programming languages
+are compared to spoken languages that have existes for thousands of years. Even a programming language like Python, 
+which is large and has more primitive terms compared to other languages, is small compared to a spoken language 
+which has far more primitve to communicate with.
+
+"API design is not a solitary activity. Show your design to as many people as you can, and take their feedback seriously. 
+Possibilities that elude your imagination may be clear to others" [Bloch, 2006].
+- I like this quote because it resonates with lots of advice I've heard over the course of my lifetime. I have been told to 
+get as many sets of eyes on my work that I can, whether it is a paper, coding project, or a piece of art I created, input from
+others will help you realize things you may have not considered. Particularly, I really liked the last sentence, I thought
+that it summed up this idea nicely.
+
+"You can't please everyone so aim to displease everyone equally" [Bloch, 2006].
```

Comment:
I really liked this quote as well (and was considering using it as one of my quotes too). It's nice and concise, but also often hard to follow. Not only is it hard to find the balance, but also letting go of your own preferences in favor of what others want.